hawkular-metrics/hawkular-metrics-wrapper.sh

```diff
@@ -91,16 +91,46 @@ fi
 sed -i "s|<!-- ##ADDITIONAL LOGGERS## -->|$additional_loggers|g" ${JBOSS_HOME}/standalone/configuration/standalone.xml
 
 # Setup the truststore so that it will accept the OpenShift cert
+KEYTOOL_COMMAND=/usr/lib/jvm/java-1.8.0/jre/bin/keytool
 HAWKULAR_METRICS_AUTH_DIR=$HAWKULAR_METRICS_DIRECTORY/auth
 mkdir $HAWKULAR_METRICS_AUTH_DIR
 pushd $HAWKULAR_METRICS_AUTH_DIR
 
-cp $KEYSTORE hawkular-metrics.keystore
-cp $TRUSTSTORE hawkular-metrics.truststore
+if [ ! -n "$KEYSTORE" ]; then
```

Comment:
The keystore can be specified via `--hmw.keystore`. If it's not specified, we load the certs from the location we expect them to be. I'm not familiar with the possible use cases, so, not sure if it's useful, but my idea would be that users would be able to override the whole keystore and truststore, in case they need more than one cert on the keystore.